hack/update-codegen.sh

```diff
@@ -33,5 +33,5 @@ bash "${CODEGEN_PKG}"/generate-groups.sh \
   all \
   sigs.k8s.io/scheduler-plugins/pkg/generated \
   sigs.k8s.io/scheduler-plugins/pkg/apis \
-  podgroup:v1alpha1 \
-  --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt
+  "podgroup:v1alpha1 capacityscheduling:v1alpha1" \
```

Comment:
resourcequota:v1alpha1? In general introduction, we can say we're doing capacity scheduling, however, to make it a concrete concept/API, let's name it resourcequota.

Reply:
`elasticquota`?

Reply:
Yes.  elasticquota

Reply:
Yes, typo...